src/plugins/ciu/ciuPlugin.test.js

```diff
@@ -0,0 +1,19 @@
+const ciuPlugin = require('./ciuPlugin');
+
+async function testCiu(message) {
+  return new Promise((resolve) => {
+    ciuPlugin({
+      command: { command: message },
+      respondWithMention: resolve,
+      handling() {},
+    });
+  });
+}
+
+it('works', async () => {
+  const output = await testCiu('ciu grid css');
+
+  expect(output).toEqual(
```

Comment:
```suggestion
  expect(output).toBe(
```
i think toBe for primitives, toEqual for objects?